src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseAgreedSettledPartAdmitDefendantLipNotificationHandler.java

```diff
@@ -64,7 +64,9 @@ private CallbackResponse notifyDefendantForPartAdmitClaimSettled(CallbackParams
 
         notificationService.sendMail(
             caseData.getRespondent1().getPartyEmail(),
-            notificationsProperties.getRespondentLipPartAdmitSettleClaimTemplate(),
+            caseData.isRespondentResponseBilingual()
```

Comment:
Please extract it to a function, let's avoid complexity within a call to another function.